src/extension.ts

```diff
@@ -255,15 +255,6 @@ async function initializeCrc(
   return hasSetupFinished;
 }
 
-// function initCommandsAndPreferences(
```

Comment:
do not leave code that has been commented. in not sued, get rid of it. we have Git to handle this.